Lib/test/test_os.py

```diff
@@ -4249,7 +4249,8 @@ def assert_stat_equal(self, stat1, stat2, skip_fields):
             self.assertEqual(stat1, stat2)
 
     # TODO: RUSTPPYTHON (AssertionError: TypeError not raised by ScandirIter)
-    @unittest.expectedFailure
+    # TODO: See https://github.com/RustPython/RustPython/issues/5190 for skip rationale 
+    @unittest.skip("skipping to avoid the unclosed scandir from squatting on file descriptors")
```

Comment:
The current change will show up as "not skipped by us" on the website. I suggest this:

```suggestion
@unittest.skip("TODO: RUSTPYTHON, avoid the unclosed scandir from squatting on file descriptors")
```